atmo/templates/atmo/jobs/mails/failed_run_alert.mail

```diff
@@ -7,7 +7,7 @@
 {% block cc %}{{ settings.DEFAULT_FROM_EMAIL }}{% endblock cc %}
 
 {% block body %}
-Your scheduled Spark job "{{ alert.run.spark_job.identifier }}" has failed
+Your scheduled Spark job "{{ alert.run.spark_job.identifier }}: {{ alert.run.spark_job.description }}" has failed
```

Comment:
I think it would be best to put the job description on the next line - it could be quite long.